### PR TITLE
[FEATURE] Add support for php:enum

### PR DESCRIPTION
**Pull request by brotkrueml**

### CODE REVIEW

**Review by linawolf — COMMENTED** [review]
Thanks, I just had some small remarks.

We will handle cases and links to enum in a follow up, I suppose?

**Comment by brotkrueml** [comment]
> Thanks, I just had some small remarks.
> 
> We will handle cases and links to enum in a follow up, I suppose?

Yup

File: tests/Integration/tests/php/enum-without-namespace/expected/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!-- content start -->
+<div class="section" id="php-enum-without-namespace-from-directive">
+    <h1>PHP Enum without namespace from directive</h1>
+
+    <dl class="php interface">
```
[the remark this answers is not in the document]
Comment (brotkrueml) [reply tests/Integration/tests/php/enum-without-namespace/expected/index.html]: Done

File: tests/Integration/tests/php/enum-without-namespace/expected/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            <span class="sig-name descname"><span
+                class="pre">SomeEnum</span></span>
+
+        </dt>
+        <dd>
+            <p>some stuff</p>
+        </dd>
+    </dl>
+
+    <dl class="php interface">
```
[the remark this answers is not in the document]
Comment (brotkrueml) [reply tests/Integration/tests/php/enum-without-namespace/expected/index.html]: Done

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
I think, this should be squash-merged

**Pull request merged**